include/diaspora.php

```diff
@@ -1682,7 +1717,7 @@ private function receive_message($importer, $data) {
 	 *
 	 * @return bool always true
 	 */
-	private function receive_participation($importer, $data) {
+	private static function receive_participation($importer, $data) {
```

Comment:
Diaspora doesn't support it by now. I wanted to implement the receiving in our next version since we are already overdue with the code freeze. There will be even more changes then, that will enhance the Diaspora protocol to transport (nearly) everything that Friendica does.